src/test/resources/pages/instructorSearchPageSearchFeedbackResponseComments.html

```diff
@@ -216,4 +216,4 @@
       </div>
     </div>
   </div>
-</div>
\ No newline at end of file
+</div>
```

Comment:
@whipermr5 the correct God mode behaviour should be _with_ a newline character in the end. I'm guessing someone did not run God mode properly (or did not use God mode) before.